mail/views.py

```diff
@@ -73,6 +74,26 @@ def post(self, request, *args, **kargs):  # pylint: disable=unused-argument
         )
 
 
+class AutomaticEmailView(GenericAPIView):
+    """
+    View class that deals with listing and editing automatic mails
+    """
+    authentication_classes = (
+        authentication.SessionAuthentication,
+        authentication.TokenAuthentication,
+    )
+    permission_classes = (permissions.IsAuthenticated, UserCanMessageLearnersPermission, )
+
+    def get(self, request, *args, **kargs):
+        """GET"""
+        emails = AutomaticEmail.objects.filter(staff_user=request.user)
+        serializer = AutomaticEmailSerializer(emails, many=True)
+        return Response(
+            status=200,
```

Comment:
you do not need to specify 200 (it is the default value for `status`), but if you want, please use the variable in `rest_framework.status` and not the number directly.